example/basic/webpack.config.js

```diff
@@ -0,0 +1,25 @@
+const { resolve } = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const WebappWebpackPlugin = require('./../../src/');
```

Comment:
it should point to the project root, not to the source folder, that is please replace ` './../../src/'` by `../../`.